tests/test_deployment.py

```diff
@@ -121,8 +121,12 @@ def test_fallback_isolation(self):
 @pytest.mark.usefixtures('bartlett_dir', 'active_config')
 class TestSetupExperiment(object):
 
-    def test_setup_creates_new_experiment(self):
+    @pytest.fixture
+    def subject(self):
```

Comment:
Why is this called subject?

Reply:
It's the _subject_ under test. I'm growing wary of inventing alternative names for test subjects in order to both re-express their original intention (which hopefully could not be done better than the actual name), but also not blow out the line length, and a few other issues. This was a bit of an experiment to see how it felt. If you found it confusing that's not a good sign. :)

Reply:
Okay, like FUT in the Pyramid tests? That makes sense. I thought it was a reference to an experimental subject.